dali/operators/reader/nvdecoder/imgproc.cu

```diff
@@ -107,22 +40,37 @@ __global__ void process_frame_kernel(
   auto src_y = static_cast<float>(dst_y) * fy + shift;
 
   // https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#tex2d-object
-  YCbCr<float> ycbcr;
-  ycbcr.y = tex2D<float>(luma, src_x, src_y);
+  vec<3, float> yuv;
+  const float scale = 255.0f/256;
+  yuv[0] = tex2D<float>(luma, src_x, src_y);
   auto cbcr = tex2D<float2>(chroma, src_x * 0.5f, src_y * 0.5f);
-  ycbcr.cb = cbcr.x;
-  ycbcr.cr = cbcr.y;
+  yuv[1] = cbcr.x;
+  yuv[2] = cbcr.y;
 
+  yuv = yuv * scale;
```

Comment:
Perhaps for readability it'd be better to define the scale right here.
```suggestion
  const float scale = 255.0f/256;
  yuv = yuv * scale;
```

Reply:
Done